client/jsonrpc.go

```diff
@@ -45,6 +45,19 @@ func (r *Request) RpcCall(result interface{}, method string, params interface{})
 	return resp.GetObject(result)
 }
 
+func (r *Request) RpcCallRaw(method string, params interface{}) ([]byte, error) {
+	req := &RpcRequest{JsonRpc: JsonRpcVersion, Method: method, Params: params, Id: genID()}
+	var resp *RpcResponse
+	err := r.Post(&resp, "", req)
+	if err != nil {
+		return nil, err
+	}
+	if resp.Error != nil {
+		return nil, resp.Error
+	}
+	return resp.GetRawBody()
```

Comment:
```suggestion
	return r.PostRaw("", req)
```

Reply:
it will not work.
we need to get Rpc.Response.Result field - that's what I did
in your way - we are getting full response `{"jsonrpc":"2.0","id":33,"result"...`
`RpcCall` returns unmarshaled `result` so that `RpcCallRaw` must return raw `result` right?

Reply:
I got another idea:
```
type RpcResponseRaw struct {
   ....
    Result json.RawMessage `json:"result'`
}
```
So you don't need to marshal again

Reply:
fixed